demo/llama_stack_mcp_agent.py

```
@@ -0,0 +1,115 @@
+    Whenever a tool is called, be sure return the Response in a friendly and helpful tone.
+    When you are asked to search the web you must use a tool.
+    """ ,
+    tools=["mcp::openshift"],
+    tool_config={"tool_choice":"auto"}
+)
+
+if args.auto:
+    user_prompts = ["""List all the namespaces in the current cluster"""]
+    session_id = agent.create_session(session_name="Auto_demo")
+
+    for i, prompt in enumerate(user_prompts):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)